.pipelines/cni/pipeline.yaml

```diff
@@ -16,12 +16,6 @@ stages:
               echo "##vso[task.setvariable variable=commitID;isOutput=true]$(echo $(make revision)-$(date "+%d%H%M"))"
               echo "##vso[task.setvariable variable=npmVersion;isOutput=true]$(make npm-version)"
               echo "##vso[task.setvariable variable=cnsVersion;isOutput=true]$(CNS_VERSION)"
-
-              if [ -z $(DROP_GZ_VERSION) ]; then

```

Comment:
will we still have a way to test the cni/cns for different release trains?

Reply:
or that will come by default with specifying the cns version?

Reply:
my expectation is that the release trains will be separate `release/v*` branches, so the pipeline only needs to test <its current codebase>. I don't need to specify a dropgz version because I am always testing CNI/CNS built from the current rev, not just dropgz built at the current rev (which had older release tags of CNI).
the pipeline caller can specify a CNI, CNS, or IPAM version to test any version skews that we think are necessary, but I have not included any now.

Reply:
What you are describing to specify a version is being deleted here. The ability to run based off a tag/rev still exists as the default value of `DROP_GZ_VERSION = ""` . 

This also exists to be able to reuse a CNI release image candidate. We do not want to generate a fresh image that could be potentially (even though it should not be) different from what we will release.